drafts/WEP0001.rst

```diff
@@ -0,0 +1,273 @@
+* people with disabilities, both visible and invisible
+* neurodivergent people
+* people with chronic illnesses or diseases
+* religious and ethnic minorities
+* age minorities (under ~21, over ~50)
+* people experiencing poverty
+* homeless and home/food insecure people
+* caregivers of children or other dependents
+* people who have experienced trauma and its aftermath (PTSD, anxiety, etc)
+* people living with or recovering from substance abuse
+
+Notably, applicants are never asked to identify which group they belong to.
+
+Selection process
+-----------------
+
+All applications will be scored by the grant review team, which should
+consist of around for reviewers. The suggested tool is Grorg, but the
+grant review team may choose different tools.
+
+Reviewers can go through all applications one by one, and score them,
+possibly with a comment. Scores, applicant names, and comments from others
+should not be shown until the reviewer has provided a score. Scores can be
+updated, but this will be visible.
+
+The scoring should be based on the overall impact that granting this person
+will have on them, the Write the Docs community, and the applicants wider
+community and country, compared to others.
+This is difficult to assess from a short form. Several guidelines are:
+
+* To what extent will letting this person attend have a further impact on the
+  community? For example, someone who essentially wrote “I will help my
+  company make more money” should likely be scored lower than someone who is
+  active in their local community.
+* The social impact of their work can be a part too. “I work to help this
+  bank I work for become richer” has less social impact than supporting someone
+  who said “I work on projects to help addicted homeless people towards recovery”,
+  and therefore the latter should be scored higher.
+* Where is this person from? If they would take a large budget, but live in an
+  area that has many other conferences closer by, they may not be a great choice.
+  So all other things being equal, we would rather spend € 1000 on an attendee
+  from Kenia than from Portland, Oregon, US.
+* Are they making a basic effort? For example, if someone doesn't even have an
+  estimate of the price of their flight without further explanation, does not
+  appear very motivated.
+* Does this conference seem like an event that would benefit them? And do
+  they have some understanding of what the conference is about?
+
+After all scores are completed, the grant review team will discuss the top
+applications, prioritised by underrepresented/marginalised groups first,
+score second, and grant them unless discussion leads to another conclusion,
+until funds run out. The review team is not required to always follow the
+score based order. This also depends on the remaining resources.
+
+Speakers will bypass the review process and are immediately accepted, except
+in exceptional circumstances.
```

Comment:
should an example of exceptional circumstances be provided?

Reply:
In theory, yes. I couldn't think of any though, but I'm always a bit careful with defining hard unbendable rules in these kind of processes.

Reply:
In theory, yes. I couldn't think of any though, but I'm always a bit careful with defining hard unbendable rules in these kind of processes.